plugins/module_utils/mysql.py

```diff
@@ -29,7 +29,7 @@
     except ImportError:
         mysql_driver = None
 
-mysql_driver_fail_msg = 'The PyMySQL (Python 2.7 and Python 3.X) or MySQL-python (Python 2.X) module is required.'
+mysql_driver_fail_msg = 'A MySQL module is required: for Python 2.7 either PyMySQL or MySQL-python, or for Python 3.X PyMySQL.  Consider setting ansible_python_interpreter to use the intended Python version.'
```

Comment:
```suggestion
mysql_driver_fail_msg = 'A MySQL module is required: for Python 2.7 either PyMySQL or MySQL-python, or for Python 3.X PyMySQL.  Consider setting ansible_python_interpreter to use the intended Python version.'
```
How about mentioning https://pypi.org/project/mysqlclient/ here as well? It's supported by the collection and is being developed by developers of PyMySQL.
(We are going to update the requirements section in https://github.com/ansible-collections/community.mysql/blob/main/plugins/doc_fragments/mysql.py.)